packages/squiggle-lang/__tests__/JS__Test.ts

```diff
@@ -1,19 +1,29 @@
 import { run } from '../src/js/index';
 
+let testRun = (x: string) => {
+  let result = run(x)
+  if(result.tag == 'Ok'){
+    return { tag: 'Ok', value: result.value.exports }
+  }
+  else {
+    return result
+  }
+}
+
 describe("A simple result", () => {
     test("mean(normal(5,2))", () => {
-        expect(run("mean(normal(5,2))")).toEqual({ tag: 'Ok', value: [ { NAME: 'Float', VAL: 5  } ] })
+        expect(testRun("mean(normal(5,2))")).toEqual({ tag: 'Ok', value: [ { NAME: 'Float', VAL: 5  } ] })
```

Comment:
Please explain your motivation for writing many test-expect clauses under one describe, instead of many describes.

Reply:
No motivation, I just add test cases when I need them to implement features xD. Would be worth refactoring the test cases. "A simple result" is not that informative.

Reply:
I would appreciate `describe` blocks being taken seriously. it's a real value add when you're sparring with the testing suite later on to have unique messages for each test case.